bin/validate-json

```diff
@@ -198,21 +210,21 @@ try {
     $urlSchema = $resolver->resolve($pathSchema, $urlData);
 
     if (isset($arOptions['--dump-schema-url'])) {
```

Comment:
Never mind, I see you just posted a commit for that at about the same time as my last comment!